### DISCUSSION TITLE
Global options

### DISCUSSION THREAD

**Discussion by hamzaelmaghari** [livewire/livewire#4904] [1. Help]
Hi devs,
Is there anything I can do to make ``protected $paginationTheme = 'bootstrap';`` global?

**Comment by zepfietje** [comment]
Hey @hoodpear!

There's currently no config option to define the pagination theme.

However, what you can do is create a custom `WithPagination` trait that extends the `WithPagination` trait from Livewire. If you define `protected $paginationTheme = 'bootstrap';` in your own trait and use that one, it should work and is nicely abstracted.

Let me know if it works for you!
